R/ArchiveBest.R

```diff
@@ -0,0 +1,111 @@
+#' @title Minimal logging object for objective function evaluations
+#'
+#' @description
+#' The [ArchiveBest] stores no data but records the best scoring evaluation
+#' passed to `$add_evals()`. The [Archive] API is fully implemented but many
+#' parameters are ignored and some methods do nothing. The archive still works
+#' with [TerminatorClockTime], [TerminatorEvals], [TerminatorNone] and
+#' [TerminatorEvals].
+#'
+#' @template param_codomain
+#' @template param_search_space
+#' @template param_xdt
+#' @template param_ydt
+#' @export
+ArchiveBest = R6Class("ArchiveBest",
```

Comment:
`ArchivePartial`? Hmm. but it should be consistent with `keep_evals == "best"`. Maybe the name is ok after all.